packages/veritone-widgets/src/widgets/SourceList/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { arrayOf, object, func, bool } from 'prop-types';
+import { SourceManagement } from 'veritone-react-common';
+import { omit } from 'lodash';
+
+import widget from '../../shared/widget';
+
+const { NullState, SourceTileView } = SourceManagement;
+
+class SourceListWidget extends React.Component {
+  static propTypes = {
+    sources: arrayOf(object).isRequired,
+    onSelectMenuItem: func.isRequired,
+    onCreateSource: func.isRequired,
+    onSelectSource: func.isRequired,
+    paginate: bool,
+    fetchData: func
+  };
+
+  constructor(props) {
```

Comment:
Pref es6 class syntax instead of constructor

Reply:
fixed